routes/campaign.php

```diff
@@ -23,6 +23,11 @@
 Route::get('/abilities/{ability}/entity-add', 'AbilityController@entityAdd')->name('abilities.entity-add');
 Route::post('/abilities/{ability}/entity-add', 'AbilityController@entityStore')->name('abilities.entity-add.save');
 
+//Ability reorder
+Route::get('/entity/{entity}/abilities/reorder', [\App\Http\Controllers\Abilities\AbilityReorderController::class, 'index'])
```

Comment:
A `entities.entity_abilities.reorder` route name would also make more sense in my opinion, keep it similar to the other entity ability route names. (and place them in the same area in the file)